### DISCUSSION TITLE
Unified Narrative for Darwinia and Crab Network

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#92] [General]
# Unified Narrative

## Darwinia Network
**Tagline**: Powering Your DApp’s Cross-Chain Capabilities.
**Brief version** : [Darwinia](https://darwinia.network/) is the community-run technology and service powering the cross-chain capabilities of decentralized applications.
**Extended version**: [Darwinia](https://darwinia.network/) is the community-run technology and service powering the cross-chain capabilities of decentralized applications. By crafting secure and efficient cross-chain messaging protocols, Darwinia is at the forefront of facilitating seamless communication between disparate blockchain networks. The newest addition to our suite of protocols is Msgport, an innovative messaging abstraction that has been successfully implemented across a wide array of mainstream smart contract platforms, broadening the potential for interoperability and enabling developers to create more versatile and connected blockchain ecosystems.

## Crab Network
Crab is a canary network with real economic value for Darwinia, similar in positioning to Polkadot's Kusama Network. It serves as a programmable cross-chain platform for DApps. Being substrate-based and EVM-compatible, it is connected to Kusama.

Special thanks to @1022ranji for helping compile this!

<details>
<summary>History of Discussion</summary>

Started in response to [this](https://github.com/orgs/dcdao/discussions/78#discussioncomment-8486731)

Hello! 

**Problem**: Since a lot has changed since Darwinia Started, the information about us on third party websites are outdated and inconsistent. 
This document aims to serve as unified location for Darwinia's important links, and narrative.

Potentially affected: Website, Docs, CoinMarketCap, CoinGecko - Feel free to add more!

## Current Status

[Website About](https://darwinia.network/#/about)
>#### Description 
>darwinia.network is a public, independent open-source resource for the Darwinia community that anyone can contribute to. We have a small core team, primarily composed of members from Itering and DCDAO, dedicated to maintaining and developing the site, with contributions from community members across the globe. All regular activity is organized through the public pull request process, ensuring that every voice can be heard and every contribution valued.

>#### Mission
>darwinia.network's mission is to be the best portal for Darwinia's growing community.

>We strive to build an easy-to-understand educational resource for Darwinia community, designed to help users become familiar with Darwinia and its key concepts. The goals include:
>- Explain Darwinia to anyone new to the technology.
>- Help new users get started with RING and Darwinia.
>- Help new developers to start building.
>- Cover updates in the Darwinia world.
>- Showcase resources created by the community.

Here we need a clear description of what Darwinia Network itself is, details about the team can come in later, mission should also focus on how does Darwinia help the overall polkadot/Blockchain ecosystem.

[Medium](https://medium.com/darwinianetwork/darwinia-highlight-january-monthly-recap-2bb209f66cad)  (Scroll down)
>Darwinia is the community-run technology and service powering the cross-chain capabilities of decentralized applications.

This is concise, but adding a bit about how Darwinia can help the cross-chain capabilities would be helpful, Although this isn't a recommendation just on the medium article, but overall generally. Albeit a good tagline! 

[Darwinia Docs](https://docs.darwinia.network/)
>Darwinia was primarily focused on blockchain and cross-chain infrastructure development. Over time, it has transformed into a community-driven decentralized entity, supported by an active community dedicated to advancing decentralized technologies. At present, Darwinia operates as a community-managed technology and service provider, concentrating on powering the cross-chain functionalities of decentralized apps.

Similarly here, we should talk more about what and how darwinia network helps esp here the focus should be directed towards the developers so using a bit technical language is good!

[CMC](https://coinmarketcap.com/currencies/darwinia-network/#News)
>Darwinia is the Web3.0 cross-chain bridge hub built on Substrate. Its major applications include Defi, NFT, Game. Darwinia is also written in the Polkadot light-paper.

>RING is the native token of Darwinia and the primary asset of the Metaverse game Evolution Land.

>Evolution Land will have 26 continents, each of which is deployed on a different public chain. Users pay RING for NFT land auction and mine resource tokens. Users play to earn from the Gamefi Farm and enjoy the fun of cross-chain trading, wars, and architecture. Evolution Land has integrated multi-chain NFTs and 10+ projects including Polygon, YGG have won the NFT land auction and build communities in Evolution Land to evolve into a more open Metaverse.

CMC talks about Evolution Land as well, and as apparent, this narrative is widely different from the rest of the locations of Darwinia.

[Coingecko](https://www.coingecko.com/en/coins/darwinia-network) is missing description and is populated with generic data.

## How to help?
I am happy to compile and unify a clear and well defined narrative for us, for me to be able to do that, I need a few people to share their thoughts and vision on what Darwinia has grown into in recent years.

 CC @hackfisher @1022ranji
 
 </details>

**Comment by 1022ranji** [comment]
I notice that this discussion extensively introduces the darwinia.network website, rather than explaining what Darwinia is. However, for third-party platforms, what's required is an introduction to the concept of Darwinia itself. The introduction text about Darwinia can be presented in two versions @nblogist:
1. Brief version (suitable for third-party platforms with a word limit): [Darwinia](https://darwinia.network/) is the community-run technology and service powering the cross-chain capabilities of decentralized applications.
2. Extended version: [Darwinia](https://darwinia.network/) is the community-run technology and service powering the cross-chain capabilities of decentralized applications. By crafting secure and efficient cross-chain messaging protocols, Darwinia is at the forefront of facilitating seamless communication between disparate blockchain networks. The newest addition to our suite of protocols is Msgport, an innovative messaging abstraction that has been successfully implemented across a wide array of mainstream smart contract platforms, broadening the potential for interoperability and enabling developers to create more versatile and connected blockchain ecosystems.

**Reply by 1022ranji** [reply]
In some contexts, a slogan might be used. Darwinia's slogan is: `Powering Your DApp’s Cross-Chain Capabilities`.

**Reply by 1022ranji** [reply]
In addition to this information, it appears that various third-party websites display different error messages that need to be corrected. Next, I will compile the sections that need to be updated/adjusted on CoinGecko (https://www.coingecko.com/en/coins/darwinia-network) into a document, making it easier for @nblogist to proceed with the next steps.

**Reply by 1022ranji** [reply]
> Here we need a clear description of what Darwinia Network itself is, details about the team can come in later, mission should also focus on how does Darwinia help the overall polkadot/Blockchain ecosystem.

The `[about darwinia.network](https://darwinia.network/#/about)` page for darwinia.network is actually an explanation of the website itself, emphasizing how the site operates, and it's not focused on introducing Darwinia Network. This is similar to the 'About' sections on the Ethereum (https://ethereum.org/en/about/) and Bitcoin (https://bitcoin.org/en/about-us) websites, which describe their operational models as community-driven. This is distinct from detailing what Darwinia Network is about. So, my suggestion would be that the 'about darwinia.network' on Darwinia Network Website doesn't need to be altered.

What I'd prefer is to correct any misinformation or outdated details that third-party platforms may have about Darwinia Network. @nblogist  cc @wuminzhe @hackfisher

**Reply by 1022ranji** [reply]
> Darwinia was primarily focused on blockchain and cross-chain infrastructure development. Over time, it has transformed into a community-driven decentralized entity, supported by an active community dedicated to advancing decentralized technologies. At present, Darwinia operates as a community-managed technology and service provider, concentrating on powering the cross-chain functionalities of decentralized apps.
> 
> Similarly here, we should talk more about what and how darwinia network helps esp here the focus should be directed towards the developers so using a bit technical language is good!

I believe the usecases section in Darwinia Docs (https://docs.darwinia.network/use-cases-d707f114326a4fed8cdc6c2b37b444b1) can provide developers with a clearer understanding of what they can achieve using Darwinia. However, if you have further ideas on how to enhance this section, feel free to discuss them with @boundless-forest.

**Reply by nblogist** [reply]
@1022ranji capabilities is a mouthful word in my opinion, how do you feel about,"features"

Description: [Darwinia](https://darwinia.network/) is the community-run technology and service powering the cross-chain _features_ of decentralized applications.
Tagline: Powering Your DApp’s Cross-Chain _Features_

**Reply by hackfisher** [reply]
I think it is better to use capabilities, at least in slogan and tagline which need to be more intuitive and easy to understand, rather than technical and precise.

**Comment by 1022ranji** [comment]
- [ ] 1. [CoinGecko](https://www.coingecko.com/en/coins/darwinia-network)
I've reviewed the information on CoinGecko and found quite a few errors/outdated information. I've compiled it into the document below for reference @nblogist : https://darwinia.notion.site/Update-CoinGecko-Darwinia-Network-Info-a5c0553839c7443faae8aaa24611ca3a
<img width="743" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/2635fa8f-b43e-416c-8cbb-5f762fa5d80e">

**Reply by nblogist** [reply]
@1022ranji I really appreciate the detailed response! :rocket:

**Comment by 1022ranji** [comment]
- [ ] 2. [CoinMarketCap](https://coinmarketcap.com/currencies/darwinia-network/)
Upon examining the details listed on CoinMarketCap, I've identified a number of inaccuracies and instances of outdated information. For your reference, I've documented these findings in the link provided below @nblogist. : https://darwinia.notion.site/Update-CoinMarketCap-Darwinia-Network-Info-d0669ba079d14067b4cfd265b5a16f96?pvs=4
<img width="568" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/665f0e2d-ce34-4295-b425-6d07fb4aea58">

**Reply by 1022ranji** [reply]
- [ ] 3. [CryptoSlate](https://cryptoslate.com/coins/darwinia-network/)
- [ ] 4. [ParachainInfo](https://parachains.info/details/darwinia)

**Reply by nblogist** [reply]
@1022ranji just want to confirm if tronscan is officially recognized, since it is mentioned on CMC but not on the CoinGecko page:
Coingecko: (Tron missing)
![image](https://github.com/dcdao/discussion/assets/35498810/ec2c0292-6be0-4c72-a6d2-16132ad1cb04)
CMC: (Tron present)
![image](https://github.com/dcdao/discussion/assets/35498810/064ec0d0-b8d2-4630-b68b-60daefa8fc50)

**Reply by nblogist** [reply]
https://www.notion.so/9617e154ec884b07a7cee9a056374e42?v=0c3e4d9f257646c486a32a0425ee3a93&ref=parachains-info

@1022ranji looks like this link is deprecated too? Pls confirm

**Reply by 1022ranji** [reply]
> @1022ranji looks like this link is deprecated too? Pls confirm

I can open this link; it's about the roadmap. You may not be able to open this link due to permission issues.
You can find the roadmap in the help center on the website: [](https://darwinia.notion.site/a76d5bdc7ad242bea991c023dfca208f?v=5fb127fc9a7a4d67a3bbc3ed50445180)
<img width="891" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/3a063aa3-f974-4599-87a2-8b39d08fea7c">
<img width="987" alt="image" src="https://github.com/dcdao/discussion/assets/80877629/19e77d27-a67a-47a7-bc36-7aac09d9bae7">

**Reply by 1022ranji** [reply]
> @1022ranji just want to confirm if tronscan is officially recognized, since it is mentioned on CMC but not on the CoinGecko page:
> Coingecko: (Tron missing)

My understanding is that there's no official or unofficial designation for explorers; all explorers are third-party DApps. If it's not listed in my draft, I might have forgot it. You have the option to add it yourself on CoinGecko.

**Reply by nblogist** [reply]
Understood, what i meant by recognized was by Darwinia, as sometime incorrect tokens get added too. Thank you so much for clarifying!

**Reply by nblogist** [reply]
Thank you for sharing the page, the one you shared is [this](https://darwinia.notion.site/c700133dd5a04c2494ceab489c75c907?v=8ff88199019d4236a3b9a190f5e7b9d5&pvs=25), which has mentioned the year 2024.
The one I shared seems to be an older one [here](https://www.notion.so/9617e154ec884b07a7cee9a056374e42?v=0c3e4d9f257646c486a32a0425ee3a93&ref=parachains-info), and doesn't mention the year 2024
I'll be sure to get this updated to the latest link as well!

**Comment by nblogist** [comment]
Updated the first comment in the discussion to be used as a reference. Closing now, next step #78

**Comment by nblogist** [comment]
![image](https://github.com/dcdao/discussion/assets/35498810/b66afea5-6ddd-4d00-994d-79f84965d3d6)
@1022ranji how would you describe the competitive edge of Darwinia, I think we should mention how it is bringing different types of chains together and helping them communicate, but there should be something more specific too that follows.

**Reply by nblogist** [reply]
Hi @1022ranji  resurfacing this :)

**Reply by 1022ranji** [reply]
> @1022ranji how would you describe the competitive edge of Darwinia, I think we should mention how it is bringing different types of chains together and helping them communicate, but there should be something more specific too that follows.

When discussing Darwinia's competitive edge, it's crucial to move beyond its previous characterization as a mere bridge hub. Darwinia has pivoted from a bridge-centric narrative towards providing a foundational cross-chain messaging infrastructure. This infrastructure is not limited to specific DApp functionalities but underpins a broader range of cross-chain capabilities, including but not limited to messaging, governance, and beyond.

Key Advantages of Darwinia's Cross-Chain Service:

### Chain-Free Empowerment

Darwinia revolutionizes DApp development, enabling developers to easily build across multiple chains, free from underlying protocol constraints, and focus on innovation and user experience.

### Composable Modules

`Darwinia Msgport` modules enhance functionality while maintaining separation from core protocols, streamlining the creation of composable, integrated DApps.

### Versatile Message Lines

`Darwinia Msgport` offers a unified, simple port to access a wide array of networks. It integrates a diverse range of messaging protocols like ORMP, LCMP, XCMP, and CCIP, catering to the ever-changing and varied needs of cross-chain applications.

(In essence, Darwinia's competitive edge lies in its ability to offer a deep, foundational layer for cross-chain interaction, marked by flexibility, developer empowerment, and a forward-looking approach to blockchain interoperability and application development.)

**Comment by nblogist** [comment]
New details submitted at Parachain.info!
